Makefile

```diff
@@ -8,7 +8,7 @@ SHELL=/bin/bash
 PREVIOUS_TAG=1.14.0
 TAG=1.15.0
 PKG=github.com/integr8ly/integreatly-operator
-TEST_DIRS?=$(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go -exec dirname {} \\; | sort | uniq")
+TEST_DIRS?=$(shell sh -c "find $(aouTOP_SRC_DIRS) -name \\*_test.go -exec dirname {} \\; | sort | uniq")
```

Comment:
is this a typo?

Reply:
It certainly is, It's already been fixed since, going to be pushed in the next commit